omod/src/main/java/org/openmrs/module/emr/page/controller/account/ManageAccountsPageController.java

```diff
@@ -1,35 +1,13 @@
 package org.openmrs.module.emr.page.controller.account;
 
-import org.apache.commons.lang.BooleanUtils;
-import org.openmrs.module.emrapi.account.AccountDomainWrapper;
 import org.openmrs.module.emrapi.account.AccountService;
 import org.openmrs.ui.framework.annotation.SpringBean;
 import org.openmrs.ui.framework.page.PageModel;
-import org.springframework.web.bind.annotation.RequestParam;
-
-import java.util.Iterator;
-import java.util.List;
 
 public class ManageAccountsPageController {
 
-    public void get(PageModel model, @SpringBean("accountService") AccountService accountService,
```

Comment:
This changes back to how this controller was before the previous commit.  So this is basically a revert on this file.